src/nodes.coffee

```diff
@@ -2274,7 +2274,14 @@ exports.StringWithInterpolations = class StringWithInterpolations extends Parens
     fragments.push @makeCode '`'
     for element in elements
       if element instanceof StringLiteral
-        fragments.push @makeCode element.value.slice(1, -1)
+        value = element.value.slice(1, -1)
```

Comment:
(Unrelated to this PR, but if you feel like it while at it, this could be changed to `value = element.value[1...-1]` :) )